### PR TITLE
EDSC-1116: Prevent action buttons on granule/collection list from covering item details

### PR DESCRIPTION
**Pull request by charleshuang80**

### CODE REVIEW

**Comment by macrouch** [comment]
screen shot 2016-07-01 at 3 10 50 pm

At 1200px width "Granules" in "39 Granules" still hasn't wrapped. It finally wrapped for me at 1180px.

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 1]
```diff
@@ -230,19 +232,30 @@ $panel-list-thumbnail-size: $panel-list-item-height;
   margin-left: 4px;
 }
 
-@media (max-width: 1400px) {
-  .panel-list-item-actions {
-    margin-left: 90px;
-  }
+// for absolute positioning in bottom right corner of panel item
```
Comment (mreese84) [inline app/assets/stylesheets/modules/panel_list.css.scss]: If this code block isn't needed anymore, we should just remove it.

**Comment by mreese84** [comment]
Charles, I'm good with this other than removing the commented out code. No need to keep whole chunks that aren't being used at all.

**Comment by mreese84** [comment]
+1

**Comment by macrouch** [comment]
Looks good

**Pull request merged**